dependencies.gradle

```diff
@@ -1,9 +1,9 @@
 dependencies {
-    compile("com.github.GTNewHorizons:GT5-Unofficial:master-SNAPSHOT:dev")
-    compile("com.github.GTNewHorizons:TinkersConstruct:master-SNAPSHOT:dev")
-    compile("com.github.GTNewHorizons:ironchest:master-SNAPSHOT:dev")
+    compile("com.github.GTNewHorizons:GT5-Unofficial:5.09.40.17:dev")
```

Comment:
Please don't! We need to get a system for fixing versions going, but please keep them on master-SNAPSHOT for now. That will ensure the versions match. I had to fix quite some compile issues when syncing up the versions between repositories! Once those crash, they will be impossible to debug!

Reply:
Pinning to master makes reproducible historical builds impossible and also lets bugs from other mods creep in.  It is a bad practice in software development.

It recently made tracking down a bug nearly impossible.

Reply:
I agree with @mitchej123 , we probably cannot keep it with `master-SNAPSHOT` anyways if we want to transition to a GTNH maven at one point

Reply:
Fixed point pinning also greatly improves build times over time - but the ability for the build script to say "you're using GT5-Unofficial:5.09.40.17 but 5.09.40.19 is available" would be nice.

As it is right now it is *impossible* to build GT5U as it was shipped in 2.1.2.0 or 2.1 because it points at HEAD for all its dependencies.